src/python/kiss_icp/datasets/ouster.py

```diff
@@ -0,0 +1,92 @@
+import os
+from typing import Optional
+
+import numpy as np
+
+
+class OusterDataloader:
+    """Ouster pcap dataloader"""
+
+    def __init__(
+        self,
+        data_dir: str,
+        meta: str,
```

Comment:
I'm not used to work with pcap files, but my biggest question is: *Is there any real use case where one have the `.pcap` file on some directory and the `.json` file somewhere else?*. In other words, is it really necessary to add another argument to the CLI if a "default" vale of the same_dir+<pcap_stem.json> should work? You get the chance to decide, since I have no authority on this matter :)

If I can, I would just extract the metadata file from the pcap file and fail if the filename we are making up does not exists. I know I'ts a bit hacky, but saves some arguments in the CLI